WordPress/WordPressTest/ReaderPostServiceTest.m

```diff
@@ -71,8 +71,9 @@ - (NSDictionary *)editorialDictionaryWithKey:(NSString *)key value:(id)value {
 #pragma mark - ReaderPostServiceRemote tests
 
 - (void)testSiteIsPrivate {
-    ReaderPostServiceRemote *remoteService = [[ReaderPostServiceRemote alloc] initWithRemoteApi:nil];
-
+    NSURL *baseURL = [NSURL URLWithString:@"http://127.0.0.1/testing"];
```

Comment:
Seems like you could do what you did here(https://github.com/wordpress-mobile/WordPress-iOS/pull/3808/files#diff-51076910955bedfe93a9523304f7c6a2R23) for this class since these three lines are repeated a few times in this test.